src/ol/render/canvas/replaygroup.js

```diff
@@ -65,22 +65,64 @@ ol.render.canvas.ReplayGroup = function(
    *        Object.<ol.render.ReplayType, ol.render.canvas.Replay>>}
    */
   this.replaysByZIndex_ = {};
+};
+ol.inherits(ol.render.canvas.ReplayGroup, ol.render.ReplayGroup);
 
-  /**
-   * @private
-   * @type {CanvasRenderingContext2D}
-   */
-  this.hitDetectionContext_ = ol.dom.createCanvasContext2D(1, 1);
+/**
+ * This methods creates a circle inside a fitting array. Points inside the
+ * circle are marked by true, points on the outside are marked with false.
+ * It uses the midpoint circle algorithm.
+ * @param {number} radius Radius.
+ * @returns {Array.<Array.<Boolean>>} an array with marked circel points.
+ * @private
+ */
+ol.render.canvas.ReplayGroup.createPixelCircle_ = function(radius) {
```

Comment:
I was thinking about it, but it is an almost 1:1 implementation of the standard algorithm. A test would be possible, though.